packages/jest-matchers/src/__tests__/utils-test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+
+'use strict';
+
+const {stringify} = require('../utils');
```

Comment:
this will fail without babel descructuring plugin that i added in one of the previous PRs